File: Word values.ts
export function wordValue(arr: string[]) {
  const alpha = "abcdefghijklmnopqrstuvwxyz";
  let result = [];
  for(const [index,element1] of arr.entries()){
    let tmp = [];
    for(const element2 of element1){
       if(alpha.indexOf(element2)!=-1){
         tmp.push((alpha.indexOf(element2)+1)*(index+1))
       }
    }
    result.push(tmp.reduce((a,b) => a+b,0))
  }
  return result
}

File: Word values.dart
List<int> wordValue(List<String> arr) {
  final alpha = "abcdefghijklmnopqrstuvwxyz";
  List<int> result = [];
  for (int index = 0; index < arr.length; index++) {
    List<int> tmp = [];
    for (int i = 0; i < arr[index].length; i++) {
      if (alpha.indexOf(arr[index][i]) != -1) {
        tmp.add((alpha.indexOf(arr[index][i]) + 1) * (index + 1));
      }
    }
    result.add(tmp.fold(0, (a, b) => a + b));
  }
  return result;
}
